src/homeform.cpp

```diff
@@ -2210,21 +2224,44 @@ void homeform::update() {
                                     settings.value(QStringLiteral("bike_resistance_offset"), 4.0).toDouble(),
                                 'f', 0));
             if (trainProgram) {
-                if (trainProgram->currentRow().lower_requested_peloton_resistance != -1) {
+                int8_t lower_requested_peloton_resistance = trainProgram->currentRow().lower_requested_peloton_resistance;
+                if (lower_requested_peloton_resistance != -1) {
+                    int8_t upper_requested_peloton_resistance = trainProgram->currentRow().upper_requested_peloton_resistance;
                     this->target_peloton_resistance->setSecondLine(
                         QStringLiteral("MIN: ") +
-                        QString::number(trainProgram->currentRow().lower_requested_peloton_resistance, 'f', 0) +
+                        QString::number(lower_requested_peloton_resistance, 'f', 0) +
                         QStringLiteral(" MAX: ") +
-                        QString::number(trainProgram->currentRow().upper_requested_peloton_resistance, 'f', 0));
+                        QString::number(upper_requested_peloton_resistance, 'f', 0));
+
+                    if (peloton_resistance < lower_requested_peloton_resistance) {
+                        this->peloton_resistance->setValueFontColor(QStringLiteral("red"));
```

Comment:
i already made a poll about this on the facebook group. The result was mixed: some people want this and some people hate this :D so i guess we should put a setting for these tiles in order to enable the color thing (disabled by default). You can put the setting in the tiles setting (maybe we can put a "color" setting for each tile too). What do you think?

Reply:
Or we can just add a new "Color" setting for each tile so they can choose this behaviour for every tile. Default values could be the current behaviour. I just want to make sure that everyone will be happy about this